apps/nextjs/src/app/aila/[id]/download/useDownloadView.ts

```diff
@@ -19,7 +20,7 @@ export function useDownloadView({
     onStart: () => null,
     lesson: lessonPlan,
     chatId: id,
-    messageId: getLastAssistantMessage(messages)?.id,
+    messageId: messages.filter((m) => m.role === "assistant").pop()?.id,
```

Comment:
Could we keep the helper but adapt it so it works with the new type? Or was there another reason to change it?